R/validation.R

```diff
@@ -692,11 +685,13 @@ validate_no_formula_duplication <- function(formula, original = FALSE) {
   check <- check_no_formula_duplication(formula, original)
 
   if (!check$ok) {
-    duplicates <- glue_quote_collapse(check$duplicates)
-
-    glubort(
-      "The following terms are duplicated on the left and right hand side ",
-      "of the `formula`: {duplicates}."
+    cli::cli_abort(
+      c(
+        "Terms must not be duplicated on the left- and right-hand side
+     of the {.arg formula}.",
```

Comment:
Indentation is 🤪 here

Reply:
oh yeah ⚡ done in [3b61e62](https://github.com/tidymodels/hardhat/pull/272/commits/3b61e62e513910d9f57d2a428e0239cd6b3b42ea)